.scrutinizer.yml

```diff
@@ -0,0 +1,8 @@
+imports:
+  - php
```

Comment:
> YAML uses indentation with one or more spaces to describe nested collections

I see no conflicts here. Of course for product style is better to use the same indentation.